packages/preview/showman/0.1.1/typst.toml

```diff
@@ -1,14 +1,15 @@
 [package]
+entrypoint = "lib.typ"
 name = "showman"
 description = "Eval & show typst code outputs inline with their source"
 repository = "https://github.com/ntjess/showman"
 keywords = ["manual", "documentation", "eval", "inline"]
 exclude = ["manual.pdf"]
 version = "0.1.1"
-entrypoint = "lib.typ"
 authors = ["Nathan Jessurun"]
 license = "Unlicense"
-compiler = "0.11.0"
+compiler = "0.10.0"
```

Comment:
Oops, this needs to change back to 0.11 before merging. I was doing some testing and didn't revert